### PR TITLE
Continue modeling VertCAS

### PR DESCRIPTION
**Pull request by SebastianGuadalupe**

### CODE REVIEW

File: models/VertCAS/VertCAS.jl [hunk 1]
```diff
@@ -83,13 +83,132 @@
 # where ``\Delta\tau=1``.
 #
 
-using NeuralNetworkAnalysis
+g = 32.2 # gravitational constant
```
Comment (schillic) [inline models/VertCAS/VertCAS.jl:86]: I think you still want to load the package.

File: models/VertCAS/VertCAS.jl [hunk 2]
```diff
@@ -83,13 +83,132 @@
...
+## accelerations (middle)
+const ACC_MIDDLE = Dict(:COC => 0.0, :DNC => -7g/24, :DND => 7g/24,
+                        :DES1500 => -7g/24, :CL1500 => 7g/24, :SDES1500 => -g/3,
+                        :SCL1500 => g/3, :SDES2500 => -g/3, :SCL2500 => g/3)
+
+## small (h, hdot0) model
+const Δτ = 1.0
+const A = [1  -Δτ; 0  1.]
+
+# We load the controllers in a dictionary with the keys beign the advisories.
```
Comment (schillic) [inline models/VertCAS/VertCAS.jl:97]:
```suggestion
# We load the controllers in a dictionary with the keys being the advisories.
```

File: models/VertCAS/VertCAS.jl [hunk 3]
```diff
@@ -103,13 +222,74 @@ end
...
+bad_states = HalfSpace([1.0, 0.0], 100.) ∩ HalfSpace([-1.0, 0.0], 100.)
+
+fig = plot(ylab="τ (time to reach horizontally)", xlab="h (vertical distance)")
+plot!(fig, bad_states, xlims=(-200, 200), ylims=(14, 26), alpha=0.3, c=:red)
+for o in out
+    plot!(fig, o, alpha=1.)
+end
+fig
+
+# ### Flowpipe resut for all choices of velocity
```
Comment (schillic) [inline models/VertCAS/VertCAS.jl:269]:
```suggestion
# ### Flowpipe result for all choices of velocity
```

**Review by schillic — APPROVED** [review]

**Review by mforets — APPROVED** [review]
Lgtm 🎉

**Comment by schillic** [comment]
I applied some missing fixes. Merging...

**Pull request merged**